tests/gtest/avifmetav1test.cc

```diff
@@ -59,10 +59,10 @@ TEST_P(AvifMetaBoxV1Test, SimpleOpaque) {
   }
   image->transformFlags = create_transform_flags;
   if (create_transform_flags & AVIF_TRANSFORM_IROT) {
-    image->irot.angle = 2;
+    image->irot.angle = 1;
   }
   if (create_transform_flags & AVIF_TRANSFORM_IMIR) {
-    image->imir.axis = 1;
+    image->imir.axis = 0;
```

Comment:
I am just curious as to why you changed `image->irot.angle` from 2 to 1 and changed `image->imir.axis` from 1 to 0.

Reply:
I do not think this is an issue. I believe `irot`/`imir` are a suboptimal way to represent the orientation. Wasting another bit in SlimHEIF to carry that pattern over is not worth it in my opinion.

Alternatively we could just forbid `irot`/`imir` combinations that are not part of the Exif->`irot`/`imir` mapping defined in the spec draft. I am fine with keeping the current implementation.

Reply:
I just realized that `flags` in FullBox is 24 bits, not 32 bits. So we have already used up all 24 bits in `flags`.

Your answer to my other comment also made me think that our encoder could forbid the `irot/imir` combinations that do not "roundtrip".